content/docs/components-and-props.md

```diff
@@ -171,11 +171,11 @@ function Avatar(props) {
 }
 ```
 
-The `Avatar` doesn't need to know that it is being rendered inside a `Comment`. This is why we have given its prop a more generic name: `user` rather than `author`.
+Le composant `Avatar` n'a pas besoin de savoir qu'il est rendu dans un composant ` Comment`. C'est pourquoi nous avons donné à ses props un nom plus générique: `user` plutôt que` author`.
 
-We recommend naming props from the component's own point of view rather than the context in which it is being used.
+Nous vous recommandons de nommer les props du point de vue du composant plutôt que de celui du contexte dans lequel il est utilisé.
 
-We can now simplify `Comment` a tiny bit:
+On peut maintenant un peu simplifier `Comment`:
```

Comment:
On peut maintenant legèrement simplifier -> Lighter to read in my opinion, that's only a writing style issue ^^

Reply:
Pour bien refléter le ton d’origine, ne pas hésiter à suivre :

« On peut maintenant simplifier un poil `Comment` : »